src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -812,6 +828,9 @@ private function copyOrMove($uuid, $destParentUuid, $userId, $webspaceKey, $loca
 
         $this->documentManager->flush();
 
+        // reload the original locale
+        $this->documentManager->find($document->getUuid(), $originalLocale);
```

Comment:
Return moved documents in the requested locale (instead of the last one that was "copied").